wetterdienst/core/scalar/values.py

```diff
@@ -24,7 +27,7 @@
 log = logging.getLogger(__name__)
 
 
-class ScalarValuesCore:
+class ScalarValuesCore(metaclass=ABCMeta):
```

Comment:
Why do you have ti changes so much in this class?
It seems that you have choosen the wrong achitecture here. 
Having a base class and sub classes sourcing from the base class would help you to avoid changes in that way. Adding a new service would mean that you have to build a new subclass on top of the base class and not changing code that affects all other services.

Reply:
Do you know how often I am running into problems when implementing a new weather service? It's really hard just to make up a perfect base class from the beginning and obviously the more weather services are implemented the harder it gets to find out the common things and how to make them be used effortlesly with new weather services. Of course when the goal is to support as many weather services on this globe as possible we will have to make some compromises but I think that's ok.